plugin/luasnip.lua

```diff
@@ -83,20 +83,10 @@ require("luasnip.config")._setup()
 -- (BufWinEnter -> lazy_load() wouldn't load any files without these).
 vim.api.nvim_create_augroup("_luasnip_lazy_load", {})
 vim.api.nvim_create_autocmd({ "BufWinEnter", "FileType" }, {
-	callback = function(event)
-		require("luasnip.loaders.from_lua")._load_lazy_loaded(event.buf)
-	end,
-	group = "_luasnip_lazy_load",
-})
-vim.api.nvim_create_autocmd({ "BufWinEnter", "FileType" }, {
-	callback = function(event)
-		require("luasnip.loaders.from_snipmate")._load_lazy_loaded(event.buf)
-	end,
-	group = "_luasnip_lazy_load",
-})
-vim.api.nvim_create_autocmd({ "BufWinEnter", "FileType" }, {
-	callback = function(event)
-		require("luasnip.loaders.from_vscode")._load_lazy_loaded(event.buf)
+	callback = function()
+		require("luasnip.loaders").load_lazy_loaded(
+			tonumber(vim.fn.expand("<abuf>"))
+		)
```

Comment:
From the docs of [`nvim_create_autocmd`](https://neovim.io/doc/user/api.html#nvim_create_autocmd()):
![image](https://github.com/L3MON4D3/LuaSnip/assets/9730330/11ddb30d-61ad-4c0b-84f8-a2bff504199c)
So you should be able to do this:
```suggestion
	callback = function(event)
		require("luasnip.loaders").load_lazy_loaded(event.buf)
```

Reply:
Ohh, yeah good thing you point that out, accidentially just undid part of #1047 there :sweat_smile: